tests/unit_tests/test_unit.py

```
@@ -40,6 +40,31 @@
 )
 
 
+def test_mztab_save(tiny_config, tmp_path):
+    file = tmp_path / "test.mztab"
+    writer = ms_io.MztabWriter(file)
+
+    writer.set_metadata(tiny_config)
+    writer.set_ms_run(["test.mgf"])
+
+    psm = psm.PepSpecMatch(
```

Ruff (0.15.6)

[error] Local variable `psm` referenced before assignment

(F823)